app/templates/components/article-meta.hbs

```diff
@@ -6,14 +6,13 @@
         <i class="ion-edit"></i> &nbsp; Edit Article
       {{/link-to}}
       &nbsp;&nbsp;
-      <button class="btn btn-sm btn-outline-danger" {{action onDeleteArticle article}} data-test-delete-article-button>
```

Comment:
I think this was accidentally removed.

Reply:
My mistake, the class names have been restored.